assets/options.js

```diff
@@ -0,0 +1,22 @@
+const customDomainField = document.getElementById('customDomain');
+
+// Saves options to chrome.storage.local.
+function saveOptions() {
+  chrome.storage.local.set({
```

Comment:
In general I would prefer `storage.sync` over `storage.local`. However according to https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/storage Firefox and Opera doesn't support `sync`. 

What about using `storage.local` as fallback if `storage.sync` is not available to maintain compatibility.